### PR TITLE
Update all expired orders to failed status

### PR DESCRIPTION
**Pull request by ravitb**
:

* Main purpose:
 
* Technical description:

* Dilemmas you faced with?

* Tests

* Documentation (Source/readme.md)

### CODE REVIEW

File: scripts/src/models/orders.ts [hunk 1]
```diff
@@ -119,6 +119,20 @@ export class Order extends CreationDateModel {
 		return query.getOne() as Promise<T | undefined>;
 	}
 
+	public static async updateUncompletedOrders<T extends Order>(): Promise<void> {
```
Comment (nitzantomer) [inline scripts/src/models/orders.ts]: What's the point of the generic constraint here? You're not using `T`.

File: scripts/src/models/orders.ts [hunk 2]
```diff
@@ -119,6 +119,20 @@ export class Order extends CreationDateModel {
...
+		const now = moment().format("YYYY-MM-DD HH:mm:ss.SSS");
+		const errorMessage = TransactionTimeout().toString();
```
Comment (nitzantomer) [inline scripts/src/models/orders.ts]: If you're looking for the message only then you can do:
```
const errorMessage = TransactionTimeout().message;
```
And then you don't need to `JSON.stringify`.  If you keep it this way, then there's no need to first use `toString()` and then `JSON.stringify`, one should be enough.

File: scripts/src/models/orders.ts [hunk 3]
```diff
@@ -119,6 +119,20 @@ export class Order extends CreationDateModel {
...
+		await Order.createQueryBuilder()
+			.update("Orders")
```
Comment (nitzantomer) [inline scripts/src/models/orders.ts]: you probably don't need the alias here

File: scripts/src/models/orders.ts (hunk 2 above)
Comment (ravitb) [reply scripts/src/models/orders.ts]: The stringify is redundent here. thanks.

**Pull request merged**